integration-test/Css.ts

```diff
@@ -1,6 +1,8 @@
 import { Properties as Properties1 } from "csstype";
 
-/** Given a type X, and the user's proposed type X, only allow keys in X and nothing else. */
+/** This file is auto-generated by truss: https://github.com/homebound-team/truss.  Any direct edits will be lost. */
```

Comment:
Super nit-picky but maybe make this a `//` comment since it's not like a "javadoc that shows up in api docs" type comment.

...also might drop the "any direct edits will be lost" b/c I think that's insinuated by the auto-generated.